### ISSUE TITLE
Ekspand/collapse info-komponent

### ISSUE THREAD

**Issue by Lillebo** [navikt/Designsystemet-old#137]
Ref. [etterspørsel på Slack](https://nav-it.slack.com/archives/C6HJFRRMY/p1579596640003800)

Bør ses i sammenheng med [Hjelpetekst](https://design.nav.no/components/hjelpetekst) og [info-variant av Alertstripe](https://design.nav.no/components/alertstripe), og den planlagte inline-varianten av Alertstripene:

<img width="587" alt="Screenshot 2020-01-21 at 10 19 35" src="https://user-images.githubusercontent.com/74510/72791401-904c2a80-3c37-11ea-9bd6-dc6615785fe8.png">

Kanskje noe àla:

<img width="201" alt="Screenshot 2020-01-21 at 10 20 37" src="https://user-images.githubusercontent.com/74510/72791487-b245ad00-3c37-11ea-834e-f46517cbd8e4.png">

**Comment by Lillebo** [comment]
Bør også ses i sammenheng med #136

**Comment by frodehansen2** [comment]
Ser bra ut. Men dersom komponenten skal brukes på infosider, ville det vært fint å kunne rendre den uten ikonet, og at en kan sette hvilken header-tag som skal brukes. På sykdom-i-familien er den som hvilken som helst annen header på samme nivå i artikkelen.

**Comment by Lillebo** [comment]
Mener du egentlig en slags variant av denne @frodehansen? https://design.nav.no/components/lesmerpanel

Vi burde sikkert se på den samtidig også. Tror uansett vi skal være litt forsiktig og ikke blande alle forskjellige behov for ekspand/collapse i en og samme komponent - hvor denne da blir nødt til å støtte hundre forskjellige "moduser". Da foretrekker jeg at vi heller rendyrker egne komponenter til spesifikke formål.

**Comment by frodehansen2** [comment]
Ja, på en måte, men vi har strukturert det med heading + content, og heading er det som trigger open/collapse. Så en slags hybrid av lesmerpanel og den du viser over.
Jeg er enig at vi ikke må legge inn for mye i en komponent, og jeg tror det kan være greit å skille mellom hva som passer i innholdssider og hva som passer i skjemaløsninger. Alt for innholdsløsninger må f.eks. ha en bra utskriftsvisning også. Vi brukte en del til på det på sykdom i familien sidene.

**Comment by randisolli** [comment]
Se også i sammenheng med #26
